geometry_msgs/Transform grasp
geometry_msgs/Transform centroid
geometry_msgs/Transform goal_centroid
---
geometry_msgs/Transform goal_ee
std_msgs/Bool success
